src/environmentd/src/http.rs

```diff
@@ -846,6 +848,7 @@ async fn init_ws(
         helm_chart_version.clone(),
         |_session| (),
         options,
+        SYSTEM_TIME.clone(),
```

Comment:
we probably don't want to do this but I haven't seen a great struct to hang this on